http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppUsageComponent.scala

```diff
@@ -51,7 +52,7 @@ object appUsageQuery extends TableQuery(new AppUsageTable(_)) {
     } yield id
 
     for {
-      res <- dbio.transaction.attempt
+      res <- dbio.transaction(TransactionIsolation.Serializable).attempt
```

Comment:
Would it make sense to do this for the other write action, `recordStop`, as well?

Reply:
I don't think we need it in `recordStop` because even if it happens, it'll just mean the same row will be updated twice with very close value...

`TransactionIsolation.Serializable` can cause deadlock, so I think we should limit use of it too

Reply:
makes sense